File: Sorting/InsertionSort/LeetCode75 Sort Color/Solution..cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        for (int i = 1; i < nums.size(); i++){
            int tmp = nums[i];
            int j;
            for (j = i; j > 0; j--){
                if (nums[j - 1] > tmp)
                    nums[j] = nums[j - 1];
                else
                    break;
            }
            nums[j] = tmp;
        }
    }
};

File: Sorting/InsertionSort/LeetCode75 Sort Color/Solution.py
class Solution:
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        for i in range(len(nums)):
            tmp = nums[i]
            for j in range(i, -1, -1):
                if 0 == j or nums[j - 1] <= tmp:
                    nums[j] = tmp
                    break
                else:
                    nums[j] = nums[j - 1]